scripts/fast-reboot

```diff
@@ -13,6 +13,9 @@ STRICT=no
 REBOOT_METHOD="/sbin/kexec -e"
 ASSISTANT_IP_LIST=""
 ASSISTANT_SCRIPT="/usr/bin/neighbor_advertiser"
+DEVPATH="/usr/share/sonic/device"
+PLATFORM=$(sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
+PLATFORM_PLUGIN="platform_plugin"
```

Comment:
The name "platform_plugin" is vague. Suggest to change to "fastboot_plugin" or similar?

Reply:
Changed to "fastboot_plugin". Please approve the changes.